js/tests/unit/scrollspy.js

```diff
@@ -231,48 +235,40 @@ $(function () {
 
   QUnit.test('should correctly select navigation element on backward scrolling when each target section height is 100%', function (assert) {
     assert.expect(5)
+    var done = assert.async()
+
+    var navLinks = ''
+    var contentDivs = ''
+    var i = 0
+
+    while (++i < 6) {
+      navLinks += '<li id="li-100-' + i + '"><a href="#div-100-' + i + '">div ' + i + '</a></li>'
+      contentDivs += '<div id="div-100-' + i + '" style="position: relative; height: 100%; padding: 0; margin: 0">div ' + i + '</div>'
+    }
+
     var navbarHtml =
         '<nav class="navbar">'
       + '<ul class="nav">'
-      + '<li id="li-100-1"><a href="#div-100-1">div 1</a></li>'
```

Comment:
Would be great if this would be on code guidelines. Like "We tend to chose readability over DRY principles in Unit tests".
BTW, I'm kind of a person who like to optimize or automatize and I see a lot of code which is not DRY in library files. I believe those optimizations can save some bytes after minification. Where I should propose those changes before actually coding? Sorry, I'm new to github.